scripts/get-packages-versions.mjs

```diff
@@ -6,6 +6,7 @@ const packagesMap = {
     '@gravity-ui/uikit': 'uikit',
     '@gravity-ui/components': 'components',
     '@gravity-ui/date-components': 'date-components',
+    '@gravity-ui/navigation': 'navigation',
```

Comment:
Add `@gravity-ui/navigation` to this line plz:
https://github.com/gravity-ui/landing/blob/main/.github/workflows/upgrade-sandbox-packages.yml#L25 

This job creates PR with lib updates every week.